workers/unity/Assets/Tests/PlaymodeTests/Performance/ComponentUpdateTests.cs

```diff
@@ -0,0 +1,51 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.TestUtils;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+
+namespace Improbable.Gdk.PlaymodeTests
+{
+    [TestFixture]
+    public class ComponentUpdateTests : MockBase
+    {
+        private const string WorkerType = "TestWorkerType";
+        private const long EntityId = 100;
+
+        [Performance, Test]
+        public void SinglePositionUpdate()
+        {
+            var markers = new[] { "GatherAllChunks", "ExecuteReplication", "Position.SendUpdates" };
+
+            var currentState = World.Step(world =>
+            {
+                world.Connection.CreateEntity(EntityId, GetEntityTemplate());
+                world.Connection.ChangeAuthority(EntityId, Position.ComponentId, Authority.Authoritative);
+            });
+
+            ActionMeasurement.Measure(() =>
+                {
+                    currentState.Step(world =>
+                    {
+                        var workerSystem = world.Worker.World.GetExistingSystem<WorkerSystem>();
+                        var entity = workerSystem.GetEntity(new EntityId(EntityId));
+                        var position = workerSystem.EntityManager.GetComponentData<Position.Component>(entity);
+
+                        position.Coords = new Coordinates(3006, 42, 2020);
+                        workerSystem.EntityManager.SetComponentData(entity, position);
+                    });
+                })
+                .ProfilerMarkers(markers)
+                .WarmupCount(100)
+                .MeasurementCount(1000)
```

Comment:
How do we choose these numbers?

Reply:
For now we need to play it by ear and vary it based on how complex the test is. More complex deserve more runs so that spikes can show up.

Looking ahead though, these should be inferred automatically a la [Google Benchmark](https://github.com/google/benchmark#runtime-and-reporting-considerations)

Reply:
For the time being I'm thinking between 30-100 measurements, with the warmup being 10% of that

Reply:
> Looking ahead though, these should be inferred automatically a la Google Benchmark

Is this work for us or work for Unity?

Reply:
Though I'd like Unity to do something about it, realistically it'll be work for us